sormas-api/src/main/java/de/symeda/sormas/api/person/PersonDto.java

```diff
@@ -883,6 +888,15 @@ public void setCitizenship(CountryReferenceDto citizenship) {
 		this.citizenship = citizenship;
 	}
 
+	@Column(length = COLUMN_LENGTH_BIG)
```

Comment:
We don't need column definitions in DTOs, move this to `Person` in sormas-backend instead and use annotation `@Column(columnDefinition = "text")`